MapboxSearch/autofill/src/androidTest/java/com/mapbox/search/autofill/AddressAutofillIntegrationTest.kt

```diff
@@ -46,15 +45,12 @@ internal class AddressAutofillIntegrationTest {
 
         val engine = createAutofillSearchEngine(
             app = APP,
-            token = TEST_ACCESS_TOKEN,
-            url = mockServer.url("").toString(),
```

Comment:
This test shouldn't pass if you don't override `url`.

IT turned out that integration tests are not run on CI. See this PR https://github.com/mapbox/mapbox-search-android/pull/182

Reply:
Done